app/middlewares/authValidator.js

```diff
@@ -15,6 +15,7 @@ exports.handle = (req, res, next) => {
   return User.findOne({ where: { email: decoded.email } })
     .then(user => {
       if (!user) next(errors.invalidAuthentication(authValidation.NOT_LOGGED_IN));
+      req.isAdmin = user.isAdmin;
```

Comment:
if you put the user in `req` you can use it in the controller directly and you don't need to find it again.

Reply:
NIce

Reply:
You are right